src/eval.jl

```diff
@@ -132,7 +132,11 @@ function evaluate_entry(session, msgid, file, line, value, repl = Base.active_re
             if isnothing(index_mime)
                 io = IOBuffer()
                 ctx = IOContext(io, [k => v for (k, v) in session.sessionparams["iocontext"]]...)
-                Base.invokelatest(show, ctx, MIME("text/plain"), first(repl_response))
+                try
+                    Base.invokelatest(show, ctx, MIME("text/plain"), first(repl_response))
+                catch exception
+                    nothing
```

Comment:
Shouldn't we notify the user that something went wrong here? Either by sending an error report to the editor, or at least displaying a warning?

Reply:
the Error and trace that repl would show still gets put out on the repl when such code (as mentioned in #49) is smuggled.
It gets handled inside `show` . So printing the error again would be redundant.

We could handle it with a warning and a Diagnostic message on the vim side. I'll take a look into how that can be handled.

Reply:
I see, sorry for missing your point. I think it's Ok to silently ignore then. Can you simply add a comment explaining why it's OK, so I don't freak out in the future when I'll have forgotten why it is OK? :)

Reply:
I think you can add the same safeguard [line 144](https://github.com/Klafyvel/REPLSmuggler.jl/pull/50/files#diff-60fedf1f907ac122cd179cc941b0a6c08297e2298fb881399bf41a3e1ee1dc84R144) when saving the output to a file. This time it should not be silent, but have a `@warn` instead.

Reply:
Forgot to add the warm , will do it in a while and push.